portfolio/src/main/webapp/index.html

```diff
@@ -88,7 +88,19 @@ <h3>Interests</h3>
 
             <br/>
             <input type="submit" />
-        </form>        
+        </form>
+        
+        
+        <h1>Translation</h1>
+        <p>Click the translate button to translate a random fact about me to Japanese.</p>
+
+
+        <button onclick="Translate();">Translate</button>
```

Comment:
It looks like this call to `Translate()` also needs to have the first letter lower-cased.

Reply:
Thank you. I forgot to change it also.